app/templates/direct-award/save-search.html

```diff
@@ -27,10 +27,31 @@
 
 <div class="single-question-page save-search-page">
   <div class="grid-row">
+    <div class="column-one-whole">
+        {% if form.name.errors or save_search_selection_error %}
+        <div class="error-summary" role="alert" aria-labelledby="saving-search-error" tabindex="-1">
+
+          <div class="dmspeak">
+            <h2 class="heading-xmedium">There was a problem saving your search</h2>
```

Comment:
I think I'd like to see how this looks, but it seems weird to have essentially a sub-heading _before_ the heading to which it relates. Maybe <h2> is not the right element. This almost feels like a flash message. If this is really the pattern from govuk then of course I'll stand corrected...

Reply:
https://govuk-elements.herokuapp.com/errors/#summarise-errors

Hey - this is the template we followed.

Reply:
Okay, yeah I suspected as much. Cheers.